sunpy/net/vso/vso.py

```diff
@@ -188,7 +195,8 @@ def search(self, *query):
 
     @classmethod
     def create(cls, queryresult):
-        return cls(list(iter_records(queryresult)), queryresult)
```

Comment:
Do we use this `iter_records` anywhere else? If not can we remove it, if so should we be sorting that as well?

Reply:
If we sort in the original method? Keyword arg to specific the type of sorting?

Reply:
No `iter_records` is not used anywhere, except it was used earlier in `create` which I changed to `iter_sort_responses()` . @nabobalis I didn't understand type of sorting. Is it like ascending or descending? Or it is just sort or not sort(keep the default)?